src/components/chart/common-chart-options.js

```diff
@@ -38,6 +38,11 @@ class CommonChartOptions {
                         return false;
                     },
                 },
+                accessibility: {
+                    keyboardNavigation: {
+                        enabled: false, // Prevents focus on legend items while keeping screen reader support
```

Comment:
Ah well done - does this disable the keyboard focus for any other chart elements that we wouldn't want it to?

Reply:
No, this will only disable keyboard focus for legends